### PR TITLE
CSHARP-3005: Create release notes for 2.11.0-beta1

### PR DESCRIPTION
**Pull request by vincentkam**

### CODE REVIEW

File: Release Notes/Release Notes v2.11.0-beta1.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  `sortKey`
+* Support for passing a hint to update commands as well as
+  `findAndModify` update and replace operations
+* Support for `allowDiskUse` on find operations
+* Support for `MONGODB-AWS` authentication using Amazon Web Services
+  (AWS) Identity and Access Management (IAM) credentials
+* Support for stapled OCSP (Online Certificate Status Protocol) (macOS only)
+
+Other new additions and updates in this beta include:
+
```
Comment (vincentkam) [inline Release Notes/Release Notes v2.11.0-beta1.md:20]: @rstam Should we include a bullet about https://jira.mongodb.org/browse/CSHARP-2724? If so, would you be able to provide a summary bullet in noun-clause form? 😺

File: Release Notes/Release Notes v2.11.0-beta1.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# .NET Driver Version 2.11.0-beta1 Release Notes
+
+This is a beta release for the 2.11.0 version of the driver.
+
+The main new features in 2.11.0-beta1 support new features in MongoDB 4.4. These features include:
```
Comment (rstam) [inline Release Notes/Release Notes v2.11.0-beta1.md]: Should this be 4.4.0?

File: Release Notes/Release Notes v2.11.0-beta1.md (hunk 1 above)
Comment (rstam) [reply Release Notes/Release Notes v2.11.0-beta1.md:20]: Oh man... we should get the docs for that in beta 1.

Maybe if I stop doing code reviews I can get at least a draft of that in today.

File: Release Notes/Release Notes v2.11.0-beta1.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  `findAndModify` update and replace operations
+* Support for `allowDiskUse` on find operations
+* Support for `MONGODB-AWS` authentication using Amazon Web Services
+  (AWS) Identity and Access Management (IAM) credentials
+* Support for stapled OCSP (Online Certificate Status Protocol) (macOS only)
+
+Other new additions and updates in this beta include:
+
+* A new target of .NET Standard 2.0
+* Support for Snappy compression on .NET Core on Windows
```
Comment (rstam) [inline Release Notes/Release Notes v2.11.0-beta1.md]: This sounds like we only support .NET Core. Should we say

    .NET Core (in addition to existing support on .NET Framework)

File: Release Notes/Release Notes v2.11.0-beta1.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+* Support for Zstandard compression on 64-bit platforms
```
Comment (DmitryLukyanov) [inline Release Notes/Release Notes v2.11.0-beta1.md]: zstadard is also available only on windows

File: Release Notes/Release Notes v2.11.0-beta1.md (hunk 1 above)
Comment (rstam) [reply Release Notes/Release Notes v2.11.0-beta1.md:20]: * New GuidRepresentationMode setting to opt-in to the new V3 GuidRepresentation mode

But maybe put this at the end of the list, not first.

**Review by rstam — CHANGES_REQUESTED** [review]
Minor comments

**Review by DmitryLukyanov — APPROVED** [review]
Suggestion about zstd and then LGTM

File: Release Notes/Release Notes v2.11.0-beta1.md (hunk 3 above)
Comment (vincentkam) [reply Release Notes/Release Notes v2.11.0-beta1.md]: Done

File: Release Notes/Release Notes v2.11.0-beta1.md (hunk 1 above)
Comment (vincentkam) [reply Release Notes/Release Notes v2.11.0-beta1.md:20]: Tweaked mildly, appended to the end of the list. @rstam, please resolve this thread if you feel it looks good!

File: Release Notes/Release Notes v2.11.0-beta1.md (hunk 4 above)
Comment (vincentkam) [reply Release Notes/Release Notes v2.11.0-beta1.md]: Ah true, updated the description

**Comment by vincentkam** [comment]
Ready for re-review

File: Release Notes/Release Notes v2.11.0-beta1.md (hunk 2 above)
Comment (vincentkam) [reply Release Notes/Release Notes v2.11.0-beta1.md]: Changed!

**Review by rstam — APPROVED** [review]
LGTM

File: Release Notes/Release Notes v2.11.0-beta1.md (hunk 2 above)
Comment (rstam) [reply Release Notes/Release Notes v2.11.0-beta1.md]: Why did this comment disappear from the Files tab?

File: Release Notes/Release Notes v2.11.0-beta1.md (hunk 3 above)
Comment (rstam) [reply Release Notes/Release Notes v2.11.0-beta1.md]: Why did this comment disappear from the Files tab?

**Pull request merged**